users/views.py

```diff
@@ -0,0 +1,66 @@
+import json
+from django.views import View
+from .models      import User
+from products.models import Product
+from django.http  import JsonResponse
+import bcrypt
+import jwt
+import django
+import re
+import my_settings
+
+class SignUpView(View):
+    def post(self , request):
+        try:
+            data = json.loads(request.body)
+            token =request.headers.get('User-Agent')
+            first_name = data["first_name"]
+            last_name  = data["last_name"]
+            birth_date = data['birth_date']
+            email      = data["email"]
+            password   = data["password"]
+
+            pattern=r'[A-Z0-9._%+-]+@[A-Z0-9,-]+\.[A-Z]{2,4}'
+            regex=re.compile(pattern,flags=re.IGNORECASE)
+            
+            if len(regex.findall(email)) == 0:
+                return JsonResponse({"message":"EMAIL_INVALID"})
+            
+            if len(password)<8 :
+                return JsonResponse({"message": "PASSWORD_LENGTH_ERROR"})
+
+            hash_password   = bcrypt.hashpw(password.encode('utf-8'),bcrypt.gensalt())
+            hash_password_d = hash_password.decode('utf-8')
+            User.objects.create(first_name=first_name,last_name=last_name,birth_date=birth_date,email=email,password=hash_password_d)
```

Comment:
```suggestion
            User.objects.create(
            
    first_name=first_name,
    last_name=last_name,
    birth_date=birth_date,
    email=email,
    password=hash_password_d
)
```
한 줄에 속성이 너무 많아서 가독성이 떨어지기 때문에, 줄로 분리해주시면 좋을 것 같습니다 :-)